.github/workflows/ci.yml

```diff
@@ -137,9 +137,10 @@ jobs:
           CI: true
 
       - name: Publish code coverage report
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v2
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
+          files: ./packages/**/coverage/lcov.info
```

Comment:
we should correctly report coverage for all packages instead of only for first one found